CHANGELOG.next.toml

```
@@ -79,3 +79,95 @@ message = "Fix cargo audit issue on criterion."
 references = ["smithy-rs#1923"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
 author = "ysaito1001"
+
+[[aws-sdk-rust]]
+message = """
+<details>
+<summary>The HTTP connector used when making requests is now configurable.</summary>
```

It technically always was configurable... but just harder.
```suggestion
<summary>The HTTP connector used when making requests is now configurable through `SdkConfig`.</summary>
```